cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -22,4 +22,7 @@ clusterStacks:
     deploy: false
   azure:
     version: v0.1.0
-    deploy: false
\ No newline at end of file
+    deploy: false
+  rook:
+    version: master
+    deploy: false
```

Comment:
With the 0.4 corssplane release, can we give stack-rook an initial 0.0.1 tag and use that as a starting point (at least for the tagged/released crossplane helm chart).  This will keep our Rook install docs in sync.

Reply:
i was planning on doing that as part of the release process, but perhaps i should do that now since it's the default.

Reply:
actually, i'll need to update all the other ones to v0.2.0 after we release, so i'll just follow up on all of those at the same time after release.